localstack/services/generic_proxy.py

```diff
@@ -146,6 +148,17 @@ def append_cors_headers(response=None):
             del headers[header]
 
 
+def http_exception_to_response(e: HTTPException):
+    """ Convert a werkzeug HTTP exception to a requests.Response object """
+    response = Response()
+    response.status_code = e.code
+    response.headers.update(dict(e.get_headers()))
+    body = e.get_body()
+    response.headers['Content-Length'] = str(len(body))
```

Comment:
nit: could `body` potentially be `None` (or any other object that does not support `len`)? Perhaps: 
```
     response.headers['Content-Length'] = str(len(str(body or '')))
```

Reply:
added the check :+1:, however `get_body()` should return a string according to the signature of `HTTPException`